evm/src/CrossChainBorrowLendMessages.sol

```diff
@@ -16,7 +16,7 @@ contract CrossChainBorrowLendMessages {
     {
         return
             abi.encodePacked(
-                header.borrower,
+                header.sender,
```

Comment:
It's the caller of the function that created the VAA. We need to encode this in the VAA for accounting purposes on the target chain, since a relayer could potentially be msg.sender on the target chain.